src/Apps/Order/Routes/Details/Components/OrderDetails.tsx

```diff
@@ -0,0 +1,173 @@
+import ShieldIcon from "@artsy/icons/ShieldIcon"
+import {
+  Box,
+  Column,
+  Flex,
+  GridColumns,
+  Image,
+  Join,
+  Message,
+  ResponsiveBox,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { type Brand, BrandCreditCardIcon } from "Components/BrandCreditCardIcon"
+
+interface OrderDetailsProps {
+  orderId: string
+}
+
+export const OrderDetails = ({ orderId }: OrderDetailsProps) => {
+  // const orderData = useFragment(ORDER_FRAGMENT, order)
+
+  return (
+    <GridColumns>
+      <Column span={[12, 8]} start={[1, 3]}>
```

Comment:
One suggestion would be to avoid any responsive stuff at all for this pass -- that will just get in the way. Better to just add a max width. (Would also avoid using `start`, since that's very rarely seen around) 

(All non-blocking!)